dabl-meta.yaml

```diff
@@ -4,16 +4,14 @@
 catalog:
     name: dabl-integration-core
     group_id: com.digitalasset
-    version: 0.5.5
+    version: 0.5.6
     short_description: The Core Pack
     description: Timer and Loopback Integrations
-    release_date: 2020-09-30
     author: Digital Asset (Switzerland) GmbH
     license: Apache-2.0
-    tags: [ integration ]
+    tags: [ integration, experimental ]
```

Comment:
For now, I want to keep the public version at `0.5.5` until the `ddit` build is complete through to publish.